### DISCUSSION TITLE
Welcome to FishyUTP Discussions!

### DISCUSSION THREAD

**Discussion by matthewshirley** [matthewshirley/FishyUTP#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

**Comment by WaliMurtaza** [comment]
How to merge a relay project with Fishnet project using your FishyUTP? I'm a beginner😅 kindly go easy on me..
